junit-jupiter-params/src/test/java/org/junit/jupiter/params/ParameterizedTestExtensionTests.java

```diff
@@ -69,8 +69,8 @@ void streamsReturnedByProvidersAreClosedWhenCallingProvide() {
 		Stream<TestTemplateInvocationContext> stream = this.parameterizedTestExtension.provideTestTemplateInvocationContexts(
 			extensionContext);
 
-		//cause the stream to be evaluated
-		stream.count();
+		// cause the stream to be evaluated
+		long unused = stream.count();
```

Comment:
Not in error-prone at least. (In my IDE at least, the warning it produces for an "unused" variable is less severe than the result of `stream.count()` not being dealt with.)

The reason behind this change, IIRC, is that error-prone has a rule which, when turned on, complains when the return value of a method that is annotated with `@CheckReturnValue` ,or is otherwise recognised to return a meaningful value, is not dealt with.

If you think this rule isn't worth keeping on for tests, please let me know and I'll happily find the rule's name and disable it.